Applications/Spire/Source/BookView/BookQuoteTableModel.cpp

```diff
@@ -103,4 +107,21 @@ void BookQuoteTableModel::on_quote_signal(const BookViewModel::Quote& quote,
     dataChanged(createIndex(index, 0), createIndex(m_size - 1,
       columnCount(QModelIndex())));
   }
+  update_market_first_indexes(index);
+}
+
+void BookQuoteTableModel::update_market_first_indexes(int index) {
```

Comment:
The order of adjectives here is incorrect, review https://dictionary.cambridge.org/grammar/british-grammar/about-adjectives-and-adverbs/adjectives-order